Bio/Affy/CelFile.py

```diff
@@ -1,10 +1,10 @@
-# Copyright 2004 by Harry Zuzan.  All rights reserved.
+# Copyright 2004 by Harry Zuzan and Adam Kurkiewicz. All rights reserved.
 # This code is part of the Biopython distribution and governed by its
 # license.  Please see the LICENSE file that should have been included
 # as part of this package.
 
 """
-Classes for accessing the information in Affymetrix cel files.
+Classes for accessing the information in Affymetrix cel files version 3 and 4
 
 Functions:
 read      Read a cel file and store its contents in a Record
```

Comment:
Sorry, I don't understand. CEL files come in 2 versions (v3 and v4). The old parser supported v3. The new parser supports v4. Both parsers support version 3 and 4, and the documentation reflects that.

Do you mean that 3 functions need to be documented for completeness:

1. read
2. readV3
3. readV4